libraries/AP_Networking/AP_Networking_Config.h

```diff
@@ -117,5 +117,5 @@
 #endif
 
 #ifndef AP_NETWORKING_SENDFILE_BUFSIZE
-#define AP_NETWORKING_SENDFILE_BUFSIZE 10000
+#define AP_NETWORKING_SENDFILE_BUFSIZE (64*512)
```

Comment:
Why this grouping instead of 32*1024?

Reply:
sector size is 512 bytes, we need multiple of sector size